client/src/HomeNavbar.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Typography } from '@mui/material';
+
+import { Link } from 'react-router-dom';
+import React from 'react';
+
+const HomeNavbar = () => {
+  return (
+    <Box sx={{ position: 'absolute' }}>
+      <Link to="/" style={{ textDecoration: 'none' }}>
+        <Typography
+          variant="h6"
+          component="div"
+          sx={{
+            marginLeft: '100px',
+            marginTop: '50px',
+            color: 'white',
+            flexGrow: 1,
+            display: { xs: 'none', sm: 'block' },
+          }}
+        >
```

Comment:
Same thing as above, it's cleaner to set margins as shown here
```suggestion
        <Typography
          variant="h6"
          component="div"
          ml="100px"
          mt="50px"
          sx={{
            color: 'white',
            flexGrow: 1,
            display: { xs: 'none', sm: 'block' },
          }}
        >
```